### ISSUE TITLE
generateUrls() does not work with universal-router/sync

### ISSUE THREAD

**Issue by ezzatron** [kriasoft/universal-router#171]
**I'm submitting a ...**

* [x] bug report
* [ ] feature request
* [ ] other (Please do not submit support requests here (below))

This is pretty easy to replicate:

```js
import UniversalRouterSync from 'universal-router/sync'
import generateUrls from 'universal-router/generateUrls'

const router = new UniversalRouter([])
const url = generateUrls(router)
```

The above code throws an error with the message `An instance of UniversalRouter is expected`.

Looking at the source it's pretty easy to understand why. The implementation of `generateUrls()` checks explicitly for `instanceof UniversalRouter`, and `UniversalRouterSync` does not inherit from `UniversalRouter`.

**Comment by frenzzy** [comment]
Thank you for the bug report! The fix is released: [v8.3.0](https://github.com/kriasoft/universal-router/releases/tag/v8.3.0) 🎉

**Comment by ezzatron** [comment]
Thanks for the quick fix!
